views/employee/single.php

```diff
@@ -3,11 +3,28 @@
  * The template for displaying CPT Example
  */
 
+use JustCoded\WP\Framework\Objects\Postmeta;
+
 /* @var \JustCoded\WP\Framework\Web\View $this */
 
+// get custom fields data
+$fields = new Postmeta();
+
 $this->extends( 'layouts/main' ); ?>
 
 	<h1><?php the_title(); ?></h1>
```

Comment:
the_title() can't be used outside of the loop (same for field values).

so this should be added inside  employee/_content template (that is where you print real data)

Reply:
replaced